bndtools.m2e/bnd.bnd

```diff
@@ -48,4 +48,6 @@ Import-Package: \
 
 -includeresource: \
 	lifecycle-mapping-metadata.xml,\
-	plugin.xml=_plugin.xml
+	plugin.xml=_plugin.xml,\
+	icons/bricks_junit.png=../bndtools.core/resources/unprocessed/icons/bricks_junit.png,\
```

Comment:
Instead of using `..`, we should use the `${workspace}` macro to reference the workspace folder.

Reply:
ah yes of course!